### PR TITLE
2 packages from yabaitechtokyo/satysfi-class-yabaitech at 0.0.6

### PR DESCRIPTION
**Pull request by zeptometer**
This pull-request concerns:
-`satysfi-class-yabaitech.0.0.6`: The yabaitech.tokyo SATySFi class file
-`satysfi-class-yabaitech-doc.0.0.6`: Document: The yabaitech.tokyo SATySFi class
 file

---
* Homepage: https://github.com/yabaitechtokyo/satysfi-class-yabaitech
* Source repo: git+https://github.com/yabaitechtokyo/satysfi-class-yabaitech.git
* Bug tracker: https://github.com/yabaitechtokyo/satysfi-class-yabaitech/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: snapshot-stable-0-0-5.opam [hunk 1]
```diff
@@ -48,10 +48,42 @@ depends: [
...
 
+  "satysfi-class-yabaitech-doc" {= "0.0.6"}
+
   "satysfi-class-yabaitech" {= "0.0.5"}
 
+  "satysfi-class-yabaitech" {= "0.0.6"}
+
   "satysfi-debug-show-value" {= "0.1.1"}
 
+  "satysfi-dist" {= "0.0.3+dev2019.02.10"}
```
Comment (na4zagin3) [inline snapshot-stable-0-0-5.opam:59]: Something went wrong.  Can you tell me the commit ids which master and origin/master branches point to?

**Comment by zeptometer** [comment]
Sorry, I found my comment was not submitted 😢

File: snapshot-stable-0-0-5.opam (hunk 1 above)
Comment (na4zagin3) [reply snapshot-stable-0-0-5.opam:59]: Ah, I see.  The script assumes `origin/master` points to this repository's master branch.

Do you mind to revert daf57d5, change the remotes so that `origin/master` points to https://github.com/na4zagin3/satyrographos-repo/tree/master, and rerun the script?

File: snapshot-develop.opam [hunk 2]
```diff
@@ -52,10 +52,42 @@ depends: [
...
 
+  "satysfi-class-yabaitech-doc" {= "0.0.6"}
+
   "satysfi-class-yabaitech" {= "0.0.5"}
 
+  "satysfi-class-yabaitech" {= "0.0.6"}
+
   "satysfi-debug-show-value" {= "0.1.1"}
 
+  "satysfi-dist" {= "0.0.3+dev2019.02.10"}
```
[the remark this answers is not in the document]
Comment (na4zagin3) [reply snapshot-develop.opam:63]: It's not expected. See https://github.com/na4zagin3/satyrographos-repo/pull/216#discussion_r515485385

**Comment by na4zagin3** [comment]
No worries!  Do you mind to fix the problem, following https://github.com/na4zagin3/satyrographos-repo/pull/216#discussion_r515485385?

File: snapshot-develop.opam [hunk 3]
```diff
@@ -52,10 +52,42 @@ depends: [
...
+  "satysfi-dist" {= "0.0.4+dev2020.04.05"}
+
+  "satysfi-dist" {= "0.0.4+dev2020.04.25"}
+
+  "satysfi-dist" {= "0.0.4+dev2020.06.07"}
+
+  "satysfi-dist" {= "0.0.5"}
+
+  "satysfi-dist" {= "0.0.5+dev2020.09.05"}
+
```
Comment (na4zagin3) [inline snapshot-develop.opam:90]:
```suggestion
```

File: snapshot-stable-0-0-5.opam [hunk 4]
```diff
@@ -48,10 +48,42 @@ depends: [
...
+  "satysfi-dist" {= "0.0.4+dev2020.04.05"}
+
+  "satysfi-dist" {= "0.0.4+dev2020.04.25"}
+
+  "satysfi-dist" {= "0.0.4+dev2020.06.07"}
+
+  "satysfi-dist" {= "0.0.5"}
+
+  "satysfi-dist" {= "0.0.5+dev2020.09.05"}
+
```
Comment (na4zagin3) [inline snapshot-stable-0-0-5.opam:86]:
```suggestion
```

File: snapshot-develop.opam [hunk 5]
```diff
@@ -52,8 +52,12 @@ depends: [
 
   "satysfi-class-yabaitech-doc" {= "0.0.5"}
 
+  "satysfi-class-yabaitech-doc" {= "0.0.6"}
+
   "satysfi-class-yabaitech" {= "0.0.5"}
 
+  "satysfi-class-yabaitech" {= "0.0.6"}
+
```
Comment (na4zagin3) [inline snapshot-develop.opam:60]:
```suggestion
  "satysfi-class-yabaitech-doc" {= "0.0.6"}

  "satysfi-class-yabaitech" {= "0.0.6"}

```

File: snapshot-develop.opam [hunk 6]
```diff
@@ -146,8 +152,12 @@ depends: [
 
   "satysfi-num-conversion" {= "0.1.1"}
 
+  "satysfi-ruby-doc" {= "0.1.1"}
+
   "satysfi-ruby-doc" {= "0.1.2"}
 
+  "satysfi-ruby" {= "0.1.1"}
+
```
Comment (na4zagin3) [inline snapshot-develop.opam:160]:
```suggestion
  "satysfi-ruby-doc" {= "0.1.2"}

```

File: snapshot-stable-0-0-5.opam [hunk 7]
```diff
@@ -48,8 +48,12 @@ depends: [
 
   "satysfi-class-yabaitech-doc" {= "0.0.5"}
 
+  "satysfi-class-yabaitech-doc" {= "0.0.6"}
+
   "satysfi-class-yabaitech" {= "0.0.5"}
 
+  "satysfi-class-yabaitech" {= "0.0.6"}
+
```
Comment (na4zagin3) [inline snapshot-stable-0-0-5.opam:56]:
```suggestion
  "satysfi-class-yabaitech-doc" {= "0.0.6"}

  "satysfi-class-yabaitech" {= "0.0.6"}

```

File: snapshot-stable-0-0-5.opam [hunk 8]
```diff
@@ -142,8 +148,12 @@ depends: [
 
   "satysfi-num-conversion" {= "0.1.1"}
 
+  "satysfi-ruby-doc" {= "0.1.1"}
+
   "satysfi-ruby-doc" {= "0.1.2"}
 
+  "satysfi-ruby" {= "0.1.1"}
+
   "satysfi-ruby" {= "0.1.2"}
```
Comment (na4zagin3) [inline snapshot-stable-0-0-5.opam:158]:
```suggestion
  "satysfi-ruby-doc" {= "0.1.2"}

  "satysfi-ruby" {= "0.1.2"}

```

**Comment by na4zagin3** [comment]
I'm sorry for my late response. Thanks for sending the package!

**Comment by zeptometer** [comment]
Sorry for the late reply, I've resolved https://github.com/na4zagin3/satyrographos-repo/pull/216#discussion_r515485385 in my local repo. Seems you've already resolved the issue in this PR, thank you for that! Is there anything I should do before this PR gets merged?

**Comment by zeptometer** [comment]
Oops, I've found that Satyristes should be fixed, (version should be `0.0.6` instead of `0.0.4`). I'll do a quick fix.

**Comment by na4zagin3** [comment]
Oops,I just forgot to merge this ticket..   Anyway, I’d be happy if you could run `satyrographos lint` on the project (you will want to opt in to the feature with setting an environment variable.)

**Comment by na4zagin3** [comment]
@zeptometer can you please let me know when it's ready? Thanks

**Comment by zeptometer** [comment]
The issue with `opam-publish` is resolved and I've updated the PR.

**Comment by zeptometer** [comment]
@na4zagin3 `satyrographos lint` gave the following error.
```
$ SATYROGRAPHOS_EXPERIMENTAL=1 satyrographos lint
Compatibility warning: You have opted in to use experimental features.
/home/zeptometer/code/yabaitech/satysfi-class-yabaitech/Satyristes:2:1: (module class-yabaitech):
Error:
  Something went wrong during working on dependency graphs.
(Failure
 "BUG: decode_path: File \"/tmp/Satyrographos2032f2lint/packages/class-yabaitech/table-of-contents/../chapter/commands.satyh\" is not found.")
Raised at Stdlib.failwith in file "stdlib.ml", line 29, characters 17-33
Called from Satyrographos_command__Lint.lint_module_dependency.verification.(fun).check_dependency.(fun) in file "src/command/lint.ml", line 282, characters 29-49
Called from Base__List.count_map in file "src/list.ml", line 408, characters 13-17
Called from Base__List.concat_map.aux in file "src/list.ml", line 636, characters 34-40
Called from Base__Result.try_with in file "src/result.ml", line 195, characters 9-15


1 problem(s) found.
```

The path `/tmp/Satyrographos2032f2lint/packages/class-yabaitech/table-of-contents/../chapter/commands.satyh` looks valid to me. Do I need to care this problem?

**Comment by na4zagin3** [comment]
@zeptometer Oops.  It was a bug in Satyrographos 0.0.2.7.  The latest develop version reports these errors.

tl;dr: They are caused by `class-yabaitech/page/page-number.satyg` requiring `base/block.satyh` and `class-yabaitech/theorem/repository.satyg` requiring `base/cross-ref.satyh`.

Anyway, I'm now working on reducing the redundant problems..

```
Compatibility warning: You have opted in to use experimental features.
/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/typeset/review-comment.satyh:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/title/page.satyh:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/table-of-contents/page.satyh:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/tmp/Satyrographos16bdc8lint/packages/class-yabaitech/table-of-contents/../chapter/commands.satyh:2.2-2.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/tmp/Satyrographos16bdc8lint/packages/class-yabaitech/table-of-contents/../chapter/../bibliography/section.satyh:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/paragraph/commands.satyh:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/page/page-number.satyg:5.2-5.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/page/footer.satyh:2.2-2.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/figure/commands.satyh:2.2-2.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/document.satyh:2.2-2.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/colophon/page.satyh:3.2-3.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/chapter/commands.satyh:2.2-2.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/tmp/Satyrographos16bdc8lint/packages/class-yabaitech/chapter/../bibliography/section.satyh:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/bibliography/section.satyh:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/theorem/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/table-of-contents/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/tmp/Satyrographos16bdc8lint/packages/class-yabaitech/table-of-contents/../chapter/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/tmp/Satyrographos16bdc8lint/packages/class-yabaitech/table-of-contents/../chapter/../table-of-contents/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/figure/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/chapter/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/tmp/Satyrographos16bdc8lint/packages/class-yabaitech/chapter/../table-of-contents/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/page/page-number.satyg:3.2-3.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/pager” (mode (Generic))

/mnt/workspace/satysfi/packages/satysfi-class-yabaitech/src/document.satyh:5.2-5.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/pager” (mode (Generic))

23 problem(s) found.
```

**Comment by zeptometer** [comment]
Thank you!

**Comment by zeptometer** [comment]
Oh sorry, I misunderstood that this PR is merged...

**Comment by zeptometer** [comment]
> tl;dr: They are caused by class-yabaitech/page/page-number.satyg requiring base/block.satyh and class-yabaitech/theorem/repository.satyg requiring base/cross-ref.satyh.

Could you elaborate on what is the problem I am supposed to fix? At least I cannot remove "@require base/block.satyh" from page-number.satyg.

**Comment by na4zagin3** [comment]
Shouldn't those `*.satyg` (e.g., page-number.satyg) packages be renamed with `*.satyh` (page-number.satyh) because they depend on packages that are unavailable in the text mode?

This is the latest lint result.

```
/home/mrty/Documents/Development/satysfi/packages/satysfi-class-yabaitech/src/page/page-number.satyg:5.2-5.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/block” (mode (Generic))

/home/mrty/Documents/Development/satysfi/packages/satysfi-class-yabaitech/src/theorem/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/home/mrty/Documents/Development/satysfi/packages/satysfi-class-yabaitech/src/table-of-contents/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/home/mrty/Documents/Development/satysfi/packages/satysfi-class-yabaitech/src/chapter/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/home/mrty/Documents/Development/satysfi/packages/satysfi-class-yabaitech/src/table-of-contents/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/home/mrty/Documents/Development/satysfi/packages/satysfi-class-yabaitech/src/figure/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/home/mrty/Documents/Development/satysfi/packages/satysfi-class-yabaitech/src/chapter/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/home/mrty/Documents/Development/satysfi/packages/satysfi-class-yabaitech/src/table-of-contents/repository.satyg:1.1-1.10:
Error: lib/dep/missing
  Missing dependency for “@require: base/cross-ref” (mode (Generic))

/home/mrty/Documents/Development/satysfi/packages/satysfi-class-yabaitech/src/page/page-number.satyg:3.2-3.11:
Error: lib/dep/missing
  Missing dependency for “@require: base/pager” (mode (Generic))

9 problem(s) found.
```

**Comment by zeptometer** [comment]
Ah, I see. Thank you for the explanation. I agree with your point and I've fixed the issue.

File: packages/satysfi-class-yabaitech-doc/satysfi-class-yabaitech-doc.0.0.6/opam [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  ["satyrographos" "opam" "install"
+   "--name" "class-yabaitech-doc"
+   "--prefix" "%{prefix}%"
+   "--script" "%{build}%/Satyristes"]
+]
+url {
+  src:
+    "https://github.com/yabaitechtokyo/satysfi-class-yabaitech/archive/0.0.6.tar.gz"
+  checksum: [
+    "md5=f6fb81a1a8d8d72fffe856b85cf08a4a"
```
Comment (na4zagin3) [inline packages/satysfi-class-yabaitech-doc/satysfi-class-yabaitech-doc.0.0.6/opam:35]:
```suggestion
    "md5=ffc96b861bd719a133f3e714fd57ece7"
```

File: packages/satysfi-class-yabaitech/satysfi-class-yabaitech.0.0.6/opam [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+  ["satyrographos" "opam" "install"
+   "--name" "class-yabaitech"
+   "--prefix" "%{prefix}%"
+   "--script" "%{build}%/Satyristes"]
+]
+url {
+  src:
+    "https://github.com/yabaitechtokyo/satysfi-class-yabaitech/archive/0.0.6.tar.gz"
+  checksum: [
+    "md5=f6fb81a1a8d8d72fffe856b85cf08a4a"
```
Comment (na4zagin3) [inline packages/satysfi-class-yabaitech/satysfi-class-yabaitech.0.0.6/opam:39]:
```suggestion
    "md5=ffc96b861bd719a133f3e714fd57ece7"
```

**Review by na4zagin3 — CHANGES_REQUESTED** [review]
MD5 hash should be updated too.

**Review by na4zagin3 — APPROVED** [review]

**Comment by zeptometer** [comment]
oops, thank you for the update!

**Comment by na4zagin3** [comment]
Thank you!!
